thewalrus/_permanent.py

```diff
@@ -21,9 +21,15 @@
   (2010), "The permanent of a square matrix", European Journal of Combinatorics, 31 (7): 1887-1891.
   <doi:10.1016/j.ejc.2010.01.010>`_
 """
+from itertools import chain
+
 import numpy as np
-from numba import jit
-from ._hafnian import hafnian_repeated
+from numba import jit, prange
+
+from scipy.special import factorial
+from scipy.linalg import sqrtm
```

Comment:
it seems like you don't need `sqrtm` anymore 
![image](https://user-images.githubusercontent.com/991946/150204063-583e40a0-1f90-4073-8e11-e80098a935c8.png)